en/docs/develop/identity-governance-rest-api.md

```diff
@@ -3,16 +3,164 @@ template: templates/swagger.html
+    
+    * Account Management Policies
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Enable Password Reset via Recovery Email |  Recovery.AdminPasswordReset.RecoveryLink | User gets notified with a link to reset password |
+    | Enable Password Reset via OTP  | Recovery.AdminPasswordReset.OTP  | User gets notified with a one time password to try with SSO login |
+    | Enable Password Reset Offline  | Recovery.AdminPasswordReset.Offline  | An OTP generated and stored in users claims |
+
+??? "Consent Management"
+
+    * Consent Management
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Controller Name | piiController | Name of the first Controller who collects the data |
+    | Contact Name | contact | Contact name of the Controller |
+    | Email Address | email | Contact email address of the Controller |
+    | Phone Number | phone | Contact phone number of the Controller |
+    | On Behalf | onBehalf | A user information (PII) Processor acting on behalf of a Controller or PII Processor |
+    | Url | piiControllerUrl | A URL for contacting the Controller |
+    | Country | addressCountry | Country of the Controller |
+    | Locality | addressLocality | Locality of the Controller |
+    | Region | addressRegion | Region of the Controller |
+    | Post Office Box Number | postOfficeBoxNumber | Post Office Box Number of the Controller |
+    | Postal Code | postalCode | Postal Code of the Controller |
+    | Street Address | streetAddress | Street Address of the Controller |
+
+??? "Analytics Engine"
+
+    * Analytics Engine Configuration
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Target Host | adaptive_authentication.analytics.receiver | Target Host |
+    | Enable Basic Authentication | adaptive_authentication.analytics.basicAuth.enabled | Enable Basic Authentication |
+    | User ID | adaptive_authentication.analytics.basicAuth.username | Target Host Secured User ID |
+    | Secret | __secret__adaptive_authentication.analytics.basicAuth.password | Target Host Secured Secret |
+    | HTTP Connection Timeout | adaptive_authentication.analytics.HTTPConnectionTimeou | HTTP Connection Timeout in milliseconds |
+    | HTTP Read Timeout | adaptive_authentication.analytics.HTTPReadTimeout | HTTP Read Timeout in milliseconds |
+    | HTTP Connection Request Timeout | adaptive_authentication.analytics.HTTPConnectionRequestTimeout | HTTP Connection Request Timeout in milliseconds |
+    | Hostname verification | adaptive_authentication.analytics.hostnameVerfier | Hostname verification. (STRICT, ALLOW_ALL) |
+
+The APIs mentioned below can be used to retrieve the above mentioned categories, connectors of the categories and properies of the categories and update the property values.
```

Comment:
connectors of the categories, properties of the categories, and update the property values.